pyanaconda/modules/payloads/installation.py

```diff
@@ -112,6 +113,9 @@ def name(self):
 
     def run(self):
         """Copy files from the driver disks to the installed system."""
+        if not conf.system.can_use_driver_disks:
+            return
```

Comment:
Could you log a message that the driver disk installation will be skipped, please?